sympy/parsing/sympy_parser.py

```diff
@@ -20,6 +20,8 @@
 from sympy.functions.elementary.miscellaneous import Max, Min
 
 
+null = ''
```

Comment:
It feels to me that using an empty string is going to make this vulnerable to (hard to debug) bugs in the future. (when for some reason `''` ends up as a key for another reason)

```python
class ParserNull:
    pass
null = ParserNull()
```
Might be a bit safer, but feel free to ignore if you don't think it is needed here.

Reply:
I understand the sentiment...but when would we ever parse a null string which, by definition, has nothing to parse? There is also a test that `null` does not appear in `local_dict` so if it ever does we could revisit it then and make the change in one place (with what you suggested). I'll make the change, however.

Reply:
On second thought, let's leave it as is -- by definition nobody should be changing nothing to something.